### ISSUE TITLE
Tmuxinator doesn't run commands on the created panes

### ISSUE THREAD

**Issue by canselcik** [tmuxinator/tmuxinator#225]
I am trying to use `tmuxinator` to start a `tmux` session with a single window containing four panes that have the following commands running. However, no matter what I change, the commands aren't executed and I just end up getting four panes with `bash` (or `zsh` in my case) running.

My yml file for this session:

```
# ~/.tmuxinator/testing.yml

name: testing
root: /Users/user/Desktop/DFS/DFS_Client/src/DistributedFS/out/artifacts/DistributedFS_jar

windows:
  - mp3:
      layout: tiled
      panes:
        - java -jar DistributedFS.jar ../../../settings.conf 127.0.0.1 1025
        - java -jar DistributedFS.jar ../../../settings.conf 127.0.0.1 1026
        - java -jar DistributedFS.jar ../../../settings.conf 127.0.0.1 1027
        - java -jar DistributedFS.jar ../../../settings.conf 127.0.0.1 1028
```

I have the default configuration generated by the `tmuxinator` setup. I am running on OSX 10.9 Mavericks, with `zsh` installed. I installed `tmuxinator` using `gem install tmuxinator`.

It seems the issue https://github.com/tmuxinator/tmuxinator/issues/187 is similar to mine. I tried setting the `pane-base-index` to both 0 and 1, but that didn't fix it.

What do you think?

**Comment by randre03** [comment]
Using tmux 2.5 have tried setting and also commenting out base-index and pane-base-index. Nothing works.

**Comment by rglassett** [comment]
@randre03 I believe this issue is duplicated by https://github.com/tmuxinator/tmuxinator/issues/485 and will be resolved by https://github.com/tmuxinator/tmuxinator/pull/542 and https://github.com/tmuxinator/tmuxinator/pull/543 if and when they are merged. I can update my fork if you'd like to test.

**Comment by randre03** [comment]
I last used tmuxinator (and tux) about 2.5-3 years ago and did not have any issues. I wonder if rolling back to earlier version would fix temporarily while waiting for #542 and #543 to be merged?

- Roger

On Sep 13, 2017, 11:13 PM -0400, Ryan Glassett <notifications@github.com>, wrote:

@randre03<https://github.com/randre03> I believe this issue is duplicated by #485<https://github.com/tmuxinator/tmuxinator/issues/485> and will be resolved by #542<https://github.com/tmuxinator/tmuxinator/pull/542> and #543<https://github.com/tmuxinator/tmuxinator/pull/543> if and when they are merged. I can update my fork if you'd like to test.

**Comment by randre03** [comment]
I’d love to give it a shot. I tried rolling back to prior version of tmuxinator and different versions of tux just in case…no help. Can’t figure out why I never had any issues with it 3-4 years ago, but now this.

- Roger

On Sep 14, 2017, 8:56 AM -0400, Roger Randall <roger@rogerrandall.com>, wrote:
I last used tmuxinator (and tux) about 2.5-3 years ago and did not have any issues. I wonder if rolling back to earlier version would fix temporarily while waiting for #542 and #543 to be merged?

- Roger

On Sep 13, 2017, 11:13 PM -0400, Ryan Glassett <notifications@github.com>, wrote:

@randre03<https://github.com/randre03> I believe this issue is duplicated by #485<https://github.com/tmuxinator/tmuxinator/issues/485> and will be resolved by #542<https://github.com/tmuxinator/tmuxinator/pull/542> and #543<https://github.com/tmuxinator/tmuxinator/pull/543> if and when they are merged. I can update my fork if you'd like to test.
